extensions/wikia/ArticleComments/classes/ArticleCommentList.class.php

```diff
@@ -1080,4 +1080,28 @@ static public function undeleteComplete($oTitle, $oUser, $reason) {
 		wfProfileOut( __METHOD__ );
 		return true;
 	}
+
+	/**
+	 * Enables commenting for an article if it is moved to a namespace that allows it
+	 * and disables it if it does not.
+	 * @param object Title $oOldTitle An object for the old article's name
+	 * @param object Title $oNewTitle An object for the new article's name
+	 * @param object User $oUser
+	 * @param integer $iOldId
+	 * @param integer $iNewId
+	 * @return bool
+	 */
+	public static function onTitleMoveComplete( Title &$oOldTitle, Title &$oNewTitle, User &$oUser, $iOldId, $iNewId ) {
+		global $wgArticleCommentsNamespaces;
+		wfProfileIn( __METHOD__ );
+
+		if ( ArticleComment::isBlog( $oNewTitle ) || in_array( $oNewTitle->getNamespace(), $wgArticleCommentsNamespaces ) ) {
```

Comment:
Don't you think that first condition is redundant? Or you're sure that Blog and Blog talk namespace are always commentable on all wikias? (that's what checks isBlog)